api/src/services/files.test.ts

```diff
@@ -69,5 +76,196 @@ describe('Integration Tests', () => {
 				expect(superCreateOne).toHaveBeenCalled();
 			});
 		});
+
```

Comment:
This is my first pass at adding some tests around the functionality for the version control. 

Please take a look and let me know if there is a way to improve these tests. It took a bit to figure out mocks for several areas. 

Here are the results of this file.
```
 ✓ src/services/files.test.ts (5)
   ✓ Integration Tests (5)
     ✓ Services / Files (5)
       ✓ createOne (2)
         ✓ throws InvalidPayloadError when "type" is not provided
         ✓ creates a file entry when "type" is provided
       ✓ uploadOne (3)
         ✓ increases a file version if primary key exists and version is null
         ✓ increases a file version if primary key exists and version is 1
         ✓ should not increase a file version if primary key does not exists

 Test Files  1 passed (1)
      Tests  5 passed (5)
   Start at  18:31:15
   Duration  2.11s (transform 905ms, setup 0ms, collect 1.85s, tests 14ms, environment 0ms, prepare 44ms)
   ```